CHANGELOG.md

```diff
@@ -4,7 +4,9 @@
 ## Unreleased
 
 * [FEATURE] Release `cortextool` via Homebrew for macOS #109
+* [FEATURE] Add new binary `e2ealerting` for measuring end to end alerting latency. #110
```

Comment:
I think we probably need to improve upon these changelogs, I have filed https://github.com/grafana/cortex-tools/issues/118 for that.